edb/lib/std/70-converters.edgeql

```diff
@@ -238,24 +238,24 @@ std::to_str(json: std::json, fmt: OPTIONAL str={}) -> std::str
 
 
 CREATE FUNCTION
-std::to_str(array: array<std::str>, delimiter: std::str) -> std::str
+std::array_join(array: array<std::str>, delimiter: std::str) -> std::str
 {
     SET volatility := 'STABLE';
     USING SQL $$
-    SELECT array_to_string("array", "delimiter");
+    SELECT array_array_join("array", "delimiter");
```

Comment:
No need for this line to be changed.

Reply:
I need to change the function name only.

Reply:
I'm not sure how to read your last comment, sorry.

Reply:
I have added the new commit. Please let me know if you need any other change.

Reply:
> I have added the new commit. Please let me know if you need any other change.

Ideally you should set up local development environment (https://edgedb.com/docs/internals/dev) and run the test suite. The PR has some errors that would make our test suite unhappy.

Reply:
Yes. I am running the test suite locally. I will add the commit once it is done.

Reply:
I have changed everything. But I am getting an SSLerror while setting up edgedb on my laptop. I am unable to run the test suite. Shall I send the new commit?

Reply:
Well, you need to update references to the functions you renamed:

1. sql.edgeql:178: to_str -> str_join
2. repl/__init__.py:689,760 to_str -> str_join